src/postamble.js

```diff
@@ -125,6 +125,10 @@ Module['then'] = function(func) {
 };
 #endif
 
+/**
```

Comment:
I am not using the Java binary because I don't have JRE. I wrote a python script to submit the code to closure compiler REST API. https://developers.google.com/closure/compiler/docs/api-tutorial3#enable-api